subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/cntr.c

```diff
@@ -12,8 +12,12 @@
 #include "hal/debug.h"
 
 #ifndef NRF_RTC
+#if defined(CONFIG_SOC_SERIES_NRF54LX)
```

Comment:
For simulation all these (in all files) `CONFIG_SOC_SERIES_NRF54LX` would need to be `CONFIG_SOC_COMPATIBLE_NRF54LX`